sdk/storage/azure-storage-blob/README.md

```diff
@@ -189,9 +188,9 @@ Once you've initialized a Client, you can choose from the different types of blo
 ## Examples
 The following sections provide several code snippets covering some of the most common Storage Blob tasks, including:
 
-* [Uploading a blob](#uploading-a-blob)
-* [Downloading a blob](#downloading-a-blob)
-* [Enumerating blobs](#enumerating-blobs)
+* [Uploading a blob](#uploading-a-blob "Uploading a blob")
+* [Downloading a blob](#downloading-a-blob "Downloading a blob")
+* [Enumerating blobs](#enumerating-blobs "Enumerating blobs")
```

Comment:
What's the rationale for adding a link title that duplicates the anchor text?

Reply:
It's a workaround for a bug that Laurent found in sphinx. Because of the same set of letters sphinx was creating a broken cross-reference link for some reason.